linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
Then let's have multiple linter configurations and name them after the assumptions they make. It's ok if you believe Babel should be the standard, but until it is, it isn't. I don't there is anything to gain in not being explicit about that.